Dockerfile

```diff
@@ -27,23 +26,23 @@ RUN mkdir /root/.ssh && \
     chmod 700 /root/.ssh/ && \
     umask 0077 && echo "$GIT_SSH_KEY" >/root/.ssh/id_rsa && \
     eval "$(ssh-agent -s)" && ssh-add /root/.ssh/id_rsa
+
+# Install npm
+RUN apt-get update && apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update && apt-get install -y nodejs npm
+
 RUN echo "installing npm packages"
 RUN npm install
```

Comment:
We only need the npm deps for running tests, which we're only doing in Travis CI, so we can remove this as well as the steps to install npm.